### DISCUSSION TITLE
Toggle device toolbar

### DISCUSSION THREAD

**Discussion by Aster89** [qutebrowser/qutebrowser#5732] [Q&A] [answered]
Is there any way to get device toolbar in devtools? I'm referring to the toolbar that allows you to change the size of the viewport to that of various devices, like smartphones and tablets, and that can be opened by clicking on the icon circled in red in [this answer](https://stackoverflow.com/a/61562028/5825294).

**Comment by toofar** [comment, marked as answer]
Either upgrade Qt or no (situation unclear), either way if you don't see it there is nothing qutebrowser can do to make it show up: https://bugreports.qt.io/browse/QTBUG-85171

**Reply by Aster89** [reply]
In `:version` I also have `Qt: 5.15.0`, so maybe I should just wait for a fix.

**Reply by The-Compiler** [reply]
Unfortunately I still don't see the device mode toggle in Qt 5.15.1. I added a comment to the Qt issue.
